src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -832,6 +832,7 @@ private static int IndexOfNonWhiteSpaceCharacter(ITextSnapshotLine line)
                         {
                             FinishExecute(executionResult.IsSuccessful);
                         }
+                        _window.SubmissionCounter++;
```

Comment:
Hmm... Just realized you can actually do that using UI, will verify/fix. Thanks!